docs/pre-release/howtos/intercepts.md

```diff
@@ -312,10 +312,6 @@ will be routed to your cluster as usual.
          Layer 5 Hostname       : dev-environment.edgestack.me
    ```
 
-   <Alert severity="info">
-      These questions were modified for clarity and given defaults in release 2.4.5.
-   </Alert>
-
```

Comment:
I put this alert under each modified intercept. Maybe we should also remove it from quick-start/qs-* and howtos/preview-urls